shared/haml/hamburger.haml

```diff
@@ -65,9 +79,16 @@
     # appear in the hamburger at mobile widths.
 
     if user_type == "teacher"
+      show_user_options = "show-mobile"
```

Comment:
Similarly, we could just have a single `show_user_options = "show-mobile"` instead of three.